### PR TITLE
Xgboost scaled learning rate

### PR DESCRIPTION
**Pull request by yan-gao-GY**
## Proposal

In the comprehensive example, add an option to allow user using scaled learning rate based on the number of clients/partitions.

### CODE REVIEW

**Comment by yan-gao-GY** [comment]
This PR has been merged with the previous (PR)[https://github.com/adap/flower/pull/2668/].
